### DISCUSSION TITLE
评测ds1000,humanevalx数据集为什么还需要另外构造一个docker环境评测，能直接集成在opencompass的项目代码里吗

### DISCUSSION THREAD

**Discussion by pzfok** [open-compass/opencompass#1372] [Q&A]
评测ds1000,humanevalx数据集为什么还需要另外构造一个docker环境评测，能直接集成在opencompass的项目代码里吗

**Comment by tonysy** [comment]
HumanEval-X includes, java/C++ etc, cannot be evaluated with python env.
